### ISSUE TITLE
CORS -allowedOrigins option not working with the OPTIONS request method

### ISSUE THREAD

**Issue by barrycaceres** [Senzing/senzing-api-server#207]
The default for Jetty's CrossOriginFilter is only to respond to GET,POST and HEAD requests.  However, it seems that browsers are now using OPTIONS instead of HEAD to pre-flight the requests and it is not returning the proper headers on response.

It is BELIEVED that adding OPTIONS to the allowed methods init parameter for the filter will fix this.

**Comment by barrycaceres** [comment]
The actual major problem here occurred with a regression when upgrading from Java 8 to Java 11 -- some dependencies that were included by default in Java 8 were not included in Java 11.

When I restored these dependencies it was able to handle the CORS pre-flight requests.

Additionally, I DID have to add `PUT` and `DELETE` to the list of allowed options via CORS since they were required for the Senzing API.  For good measure I added `PATCH` and `OPTIONS` as well.

This seems to work fine and fixes the error handling issue as well that was described in Issue #208 which is now closed as a result.  I will close this issue once the release is ready.
